### DISCUSSION TITLE
Pass content in data-tip of tooltip

### DISCUSSION THREAD

**Discussion by FairouzBF** [saadeghi/daisyui#3297] [Q&A]
Hello, i've been experimenting with daisy ui and wanted to use the tooltip to display a rather long message. For some reason it's only displaying the first word of the sentence, i tried wraping the whole thing in a <p> but still not working. If you have any idea what's i'm doing wrong, or if you have another solution, i would be grateful :)

Here is my code:
//in my Tooltip.html.twig
```
<div class="relative inline-block group">
	<div class="tooltip tooltip-bottom" data-tip={{content|raw}}>
		<span class="text-secondary hover:text-primary focus:outline-none">
            {{ ux_icon('mingcute:light-fill', { class: 'h-5 w-5' }) }}
        </span>
	</div>
</div>

```
//in my index.html.twig
```
{% set tooltipContent =
	'Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
	Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, 
	when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
	It has survived not only five centuries, but also the leap into electronic typesetting, 
	remaining essentially unchanged.'
%}

{% block content %}
	<div class="container-xxl flex-grow-1 container-p-y" data-loading-states>
		<div class="flex items-center justify-center space-x-7">
			{% include 'components/Tooltip.html.twig' with {
				content: tooltipContent|e
			} %}
		</div>
		<div id="tableDateContainer">
			...
		</div>
	</div>
{% endblock %}
``` 
//result
![image](https://github.com/user-attachments/assets/70cbaca5-3dbf-497e-940b-4d4b0a59ccd7)

**Comment by saadeghi** [comment]
Can you share a Tailwind Play link where I can see the challenge? (expected behavior vs current behavior)  
I don't want to setup a whole project just to run a twig file.

**Reply by FairouzBF** [reply]
I actually found the solution, or rather the mistake i was making 🥲 i forgot the quotes around {{content|e}}, sorry for the waste of time LOL
